quickstart.py

```diff
@@ -83,19 +82,24 @@ def missingDependencies():
   missing = []
   try:
     from concurrent.futures import Future
+    assert Future
```

Comment:
Ahh... ok. What I did was use flake8 which uses pyflakes under the hood, but lets you add `# noqa` markers. I suppose modules are truthy so this will work.